_posts/2015-09-08-code-talks.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: "ePages talks REST API"
+date: "2015-09-08 08:04:00"
+icon: code
+tags: misc
+categories: tech-stories conferences api
+authors: ["Birgit"]
+---
+
+Have you signed up for `code.talks` ?
```

Comment:
I would add some more infos about the Code talks: Describe in 1 or 2 sentences and add the dates and place. I think linking to the website is a good idea!